### PR TITLE
Jobs API Quickstarts

### PR DESCRIPTION
**Pull request by rochabr**
# Description

Including SDK and HTTP quickstart examples in Go for the Jobs API.

## Issue reference

We strive to have all PRs being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1026 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] The quickstart code compiles correctly
* [X] You've tested new builds of the quickstart if you changed quickstart code
* [X] You've updated the quickstart's README if necessary
* [X] If you have changed the steps for a quickstart be sure that you have updated the automated validation accordingly. All of our quickstarts have annotations that allow them to be executed automatically as code. For more information see [mechanical-markdown](https://github.com/dapr/mechanical-markdown#readme). For user guide with examples see [Examples](https://github.com/dapr/mechanical-markdown/tree/main/examples#mechanical-markdown-by-example).

### CODE REVIEW

File: jobs/go/http/README.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+```bash
+dapr run --app-id job-service --app-port 6200 --dapr-http-port 6280 -- go run .
+```
+
+2. On a new terminal window, schedule the `R2-D2` Job using the Jobs API.
+
+```bash
+curl -X POST \
+  http://localhost:6280/v1.0-alpha1/jobs/jobforjabba \
```
Comment (msfussell) [inline jobs/go/http/README.md:86]: Should this be the job?
```suggestion
  http://localhost:6280/v1.0-alpha1/jobs/R2D2 \
```

File: jobs/go/http/README.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        },
+        "dueTime": "2s"
+    }'
+    ```
+
+Back at the `job-service` app terminal window, the output should be:
+
+```text
+== APP - job-app == Received job request...
+== APP - job-app == Starting droid: R2-D2
```
Comment (msfussell) [inline jobs/go/http/README.md:101]: Isn't this the name of the job that is started?
```suggestion
== APP - job-app == Starting droid: R2-D2
```

Comment (rochabr) [reply jobs/go/http/README.md:101]: you are right, this was a bad copy/paste

File: jobs/go/http/dapr.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+version: 1
+apps:
+  - appDirPath: ./job-service/
+    appID: job-service
+    appPort: 6200
```
Comment (paulyuk) [inline jobs/go/http/dapr.yaml:5]: please make sure your `sdk` variant does not use same *exact ports.

File: jobs/go/http/job-scheduler/job-scheduler.go [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+	if res.StatusCode != http.StatusNoContent {
+		log.Fatalf("failed to register job event handler. status code: %v", res.StatusCode)
+	}
+
+	defer res.Body.Close()
+
+	fmt.Println("Job Scheduled:", jobName)
+
+	time.Sleep(5 * time.Second)
```
Comment (paulyuk) [inline jobs/go/http/job-scheduler/job-scheduler.go:69]: consider removing these.  but for now if stable leave them.  I think it confuses to have sleeps here in code, also in MMD readme.

File: jobs/go/sdk/dapr.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+version: 1
+apps:
+  - appDirPath: ./job-service/
+    appID: job-service
+    appPort: 6200
```
Comment (paulyuk) [inline jobs/go/sdk/dapr.yaml:5]: please make all these ports slightly unique from http/dapr.yaml

**Review by paulyuk — CHANGES_REQUESTED** [review]
please just unique-ify ports in /sdk/* version of dapr.yaml (and matching go file if needed).  then we're good.

**Review by paulyuk — APPROVED** [review]
LGTM.  thank you for this quickstart @rochabr !

**Comment by paulyuk** [comment]
Golang check fails only on SDK, which you warned me and we knew about.  Please collab with @cicoyle here why this is happening.

Meanwhile I will approve and merge change so we can make progress using Http and partial sdk.  Nice work here!

**Comment by marcduiker** [comment]
@holopin-bot @rochabr Thank you Fernando! 🙏

**Pull request merged**